webpack.config.js

```diff
@@ -107,15 +104,39 @@ const webpackConfig = {
 		chunkFilename: '[name].[chunkhash].min.js', // ditto
 		devtoolModuleFilenameTemplate: 'app:///[resource-path]',
 	},
+	optimization: {
+		splitChunks: {
+			chunks: 'all',
+			name: isDevelopment,
+			cacheGroups: {
+				tinymce: {
```

Comment:
Why is this needed? `tinymce` is used by 2 bundles (`post-editor` and `woocommerce`), so it should be split automatically into a different chunk, right?

Reply:
That's what we expected, but it wasn't happening on its own. Try removing it and then look at analyze-bundles output.